.github/workflows/update_ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
       - name: Run update
         working-directory: pulp_container
         run: |
-          .github/workflows/scripts/update_ci.sh
+          ../plugin_template/scripts/update_ci.sh
```

Comment:
The changes in this file are the result of running `plugin_template`'s `./plugin-template pulp_container --github`.  Are they good, bad, or benign; should I manually revert them?

Reply:
I believe these are good. See https://github.com/pulp/plugin_template/commit/fa6c951e5645513b6abae6a2a04f8b0e613ec599.